js/data.js

```diff
@@ -2,7 +2,22 @@
 
 var data = {
   view: 'entry-form',
-  entries: [],
+  entries: [
+    {
+      title: 'Ada Lovelace',
+      image: './images/placeholder-image-square.jpg',
+      notes: `Augusta Ada King, Countess of Lovelace was an English mathmatician and writer, chiefly known for her
+      work on Charles Babbage's proposed mechanical general-purpose computer, the Analytical Engine.`,
+      nextEntryID: 1
+    },
+    {
+      title: 'JavaScript',
+      image: './images/placeholder-image-square.jpg',
+      notes: `JavaScript, often abbreviated as JS, is a programming language that conforms to the ECMAScript
+              specification.`,
+      nextEntryID: 2
+    }
+  ],
   editing: null,
   nextEntryId: 1
 };
```

Comment:
In the `data.js` file you should be reading the data from `localStorage` and have an event listener for 'beforeunload` to save the data back to local storage when the page is closed. Refer to the `javascript-local-storage` exercise for reference.